doc/api/n-api.md

```diff
@@ -1393,51 +1393,51 @@ The JavaScript Number type is described in
 [Section 6.1.6](https://tc39.github.io/ecma262/#sec-ecmascript-language-types-number-type)
 of the ECMAScript Language Specification.
 
-#### *napi_create_string_utf16*
+#### *napi_create_string_latin1*
 <!-- YAML
 added: v8.0.0
 -->
 ```C
-napi_status napi_create_string_utf16(napi_env env,
-                                     const char16_t* str,
-                                     size_t length,
-                                     napi_value* result)
+NAPI_EXTERN napi_status napi_create_string_latin1(napi_env env,
+                                                  const char* str,
+                                                  size_t length,
+                                                  napi_value* result);
 ```
 
 - `[in] env`: The environment that the API is invoked under.
-- `[in] str`: Character buffer representing a UTF16-LE-encoded string.
-- `[in] length`: The length of the string in two-byte code units, or -1 if
-it is null-terminated.
+- `[in] str`: Character buffer representing a latin1-encoded string.
```

Comment:
Nit: I’d spell it as Latin-1 and/or say that it’s ISO-8859-1, I think most developers know it under that latter name.